python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -208,7 +208,7 @@ class RandomForestClassifier(BaseRandomForestModel,
         If 'auto' then max_features=1/sqrt(n_features).
         If 'sqrt' then max_features=1/sqrt(n_features).
         If 'log2' then max_features=log2(n_features)/n_features.
-    n_bins : int (default = 8)
+    n_bins : int (default = 32)
```

Comment:
Maybe add a line to the comment that - "for large problems, particularly those with highly-skewed input data, increasing the number of bins may improve accuracy"?

Reply:
Done